Makefile

```diff
@@ -62,10 +68,25 @@ $(RECIPES): %:
 	./scripts/package_build.py $(FLAGS) "$(@)"
 
 push: %:
-	rsync --rsync-path /opt/bin/rsync \
+	if ! rsync --rsync-path /opt/bin/rsync \
 	      --archive --verbose --compress --delete \
 	      build/repo/ \
-	      root@"$(HOST)":~/.cache/toltec/
+	      root@"$(HOST)":~/.cache/toltec/; then \
+		echo "Failed to push. Make sure rsync is installed on your reMarkable."; \
+	fi
+
+$(RECIPES_PUSH): SHELL:=/bin/bash
+$(RECIPES_PUSH): %:
+	source package/$(@:%-push=%)/package; \
+	if ! rsync --rsync-path /opt/bin/rsync \
+	      --archive --verbose --compress --delete --ignore-times \
```

Comment:
not sure `--delete` should be passed here, it makes more sense with repo (deletes everything not in repo). what do you think? (it was a surprise when i ran this and a bunch of files got deleted)

Reply:
Good point! I agree. (I just copied the rsync command from `push` without thinking about it)